docs/source/howto/workflows.rst

```diff
@@ -20,6 +20,172 @@ Running a predefined workflow
 
 `#3992`_
 
+The first step to running a predefined workflow is loading the work function or work chain class that defines the workflow you want to run.
+This can of course be achieved by importing the function or class from its respective module:
+
+.. code-block:: python
+
+    from aiida.workflows.arithmetic.add_multiply import add_and_multiply              # Import the add_and_multiply work function
+    from aiida.workflows.arithmetic.multiply_add import MultiplyAddWorkChain       # Import the MultiplyAddWorkChain WorkChain
+
+However, a more robust way of loading a workflow is using the ``WorkflowFactory``, for example:
+
+.. code-block:: python
+
+    from aiida.plugins import WorkflowFactory
+    add_and_multiply = WorkflowFactory('arithmetic.add_multiply')
+    MultiplyAndAddWorkChain = WorkflowFactory('arithmetic.multiply_add')
+
+The input argument of the ``WorkflowFactory`` is a string called the *entry point*.
+For each workflow, the entry point usually starts with with the name of the plugin package (``arithmetic``), followed by a dot and the string identifying the workflow (``multiplyadd``).
+Using the ``WorkflowFactory`` has the advantage that the entry point will not change when the packages or plugins are reorganised, and hence your code is less likely to break when updating AiiDA or the plugin that supplies the workflow.
+
+The list of installed plugins can be easily accessed via the verdi CLI:
+
+.. code-block:: console
+
+    $ verdi plugin list
+
+To see the list of ``WorkChain`` entry points, simply use:
+
+.. code-block:: console
+
+    $ verdi plugin list aiida.workflows
+
+By further specifying the entry point of the workflow, you can see its description, inputs, outputs and exit codes:
+
+.. code-block:: console
+
+    $ verdi plugin list aiida.workflows arithmetic.multiplyadd
+
+.. note::
+
+    Using the ``WorkflowFactory`` is a good paradigm to get used to as the entry point names are unique identifiers that are supported in many parts of AiiDA's API, such as the ``verdi plugin list`` command shown above.
```

Comment:
Again, the documented way to "import" workflows is using the ``WorkflowFactory``. As long as we properly motivate it once, I don't think there is much more *convincing* that we need to do. If users decide to use direct imports instead, they are employing a non-documented way of using AiiDA and may need to live with potential downsides of that approach. That means we should completely remove this sentence.